sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -69,6 +80,7 @@ export const INSUFFICIENT_APPLICATION_SEARCH_PARAMS =
 
 @Injectable()
 export class ApplicationService extends RecordDataModelService<Application> {
+  private readonly studentRepo: Repository<Student>;
```

Comment:
Please use the already-in-place method `getStudentById` from the `SudentService`. If needed this method can be shared from a more generic library to be reused.

Reply:
<img width="811" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/4db91dda-2d5b-48c7-9443-e60c27566834">

I think we already have it shared.